src/pages/fullscreen/playback/index.vue

```diff
@@ -95,15 +95,13 @@ export default Vue.extend({
       this.setBackdrop({ hash: this.backdropHash });
     });
   },
-  activated() {
+  mounted() {
     this.setAppBarOpacity({ opaqueAppBar: false });
     this.setBackdropOpacity({ newOpacity: 0.5 });
     this.setMinimized({ minimized: false });
-  },
-  mounted() {
     this.swiper = (this.$refs.playbackSwiper as Vue).$swiper as Swiper;
```

Comment:
I'd put the old mounted command before the old activated method: in the lifecycle mounted comes before activated